docs/what_is_data_science/index.md

```diff
@@ -1,3 +1,23 @@
 # What is Data Science?
 
-> A page describing what data science is.
\ No newline at end of file
+Quoting the [DDaT Capability Framework](https://ddat-capability-framework.service.gov.uk/data-scientist.html):
+
+> Data science is a broad and fast-moving field spanning maths, statistics, software engineering and communications. Data scientists will often work as part of a multidisciplinary team, using data and analytics to inform and achieve organisational goals. 
+
+Specifically, this means that often data scientists use programming languages like Python and R (though there are others too!) to solve problems within the business, or inform about courses of action, working with other colleagues such as analysts and data engineers, in particular. 
+
+Some examples of the kinds of problems data science can help with are:
+
+- **getting more value out of unstructured data** (e.g. text, images, audio) through "Natural Language Processing", Neural Networks and recently "Large Language Models" (e.g. ChatGPT).
```

Comment:
maybe adding links to "natural language processing" (like https://www.ibm.com/topics/natural-language-processing#:~:text=Natural%20language%20processing%20(NLP)%20refers,same%20way%20human%20beings%20can. ) neural networks (https://news.mit.edu/2017/explained-neural-networks-deep-learning-0414) and large language models (https://www.techtarget.com/whatis/definition/large-language-model-LLM#:~:text=A%20large%20language%20model%20(LLM,generate%20and%20predict%20new%20content.) Also, either put all three in quotation marks or none of them (neural networks is currently not in quotes)

Reply:
What I wanted to do was to have links to our own projects which use these techniques - when we've added pages for the NHS Reviews work for example, or whatever other work fits into the above, we can add links.